test/unitTests/test-reasoning-layer-lib/test-elements-reasoning-layer.xspec

```diff
@@ -146,7 +146,7 @@
     </x:scenario>
     
     
-    <x:scenario
+<!--    <x:scenario
```

Comment:
And this test?

Reply:
if the rule for Enumerations will be removed this test will be removed as well